.env.example

```diff
@@ -16,11 +20,10 @@ LIVEKIT_URL=wss://my-livekit-project.livekit.cloud
 # S3_REGION=
 
 # PUBLIC
-NEXT_PUBLIC_LK_TOKEN_ENDPOINT=/api/token
-#NEXT_PUBLIC_LK_RECORD_ENDPOINT=/api/record
-
 # Uncomment settings menu when using a LiveKit Cloud, it'll enable Krisp noise filters.
 # NEXT_PUBLIC_SHOW_SETTINGS_MENU=true
+# NEXT_PUBLIC_CONN_DETAILS_ENDPOINT=
```

Comment:
I think we should still fill at least this value in, as it won't change for users under normal circumstances

Reply:
Good catch. I would rather remove it. We set it to a hardcoded default value in the code. It just adds noise to the .env file.